static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -230,35 +231,51 @@ const canShow = async () => {
     }
 };
 
-const show = () => import(
-    /* webpackChunkName: "guardian-braze-components" */ '@guardian/braze-components'
-    )
-    .then((module) => {
-        const container = document.createElement('div');
+const show = () => Promise.all([
+    import('react-dom'),
+    import('@emotion/core'),
+    import('@emotion/cache'),
+    import(/* webpackChunkName: "guardian-braze-components" */ '@guardian/braze-components')
+]).then((props) => {
```

Comment:
`@emotion/core` and `@emotion/cache` moduels are only used in braze. As Braze doesn't nessesarly always show a banner, it is best we dynamically load these modules to reduce bundle size

Reply:
This sounds reasonable 👍  Would be interesting to see the performance impact of this in terms of how long the banner takes to show. Right now I think `@emotion/core` and `@emotion/cache` are part of the bundle already since they're a dependency of `@guardian/automat-modules` as they're used by `mountDynamic` (used [here](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/commercial/contributions-service.js#L290) for example).